proto/prysm/v1alpha1/wrapper/blinded_beacon_block_bellatrix.go

```diff
@@ -0,0 +1,315 @@
+package wrapper
+
+import (
+	ssz "github.com/ferranbt/fastssz"
+	"github.com/pkg/errors"
+	types "github.com/prysmaticlabs/eth2-types"
+	enginev1 "github.com/prysmaticlabs/prysm/proto/engine/v1"
+	eth "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/block"
+	validatorpb "github.com/prysmaticlabs/prysm/proto/prysm/v1alpha1/validator-client"
+	"github.com/prysmaticlabs/prysm/runtime/version"
+	"google.golang.org/protobuf/proto"
+)
+
+var (
+	_ = block.SignedBeaconBlock(&bellatrixSignedBlindedBeaconBlock{})
+	_ = block.BeaconBlock(&bellatrixBlindedBeaconBlock{})
+	_ = block.BeaconBlockBody(&bellatrixBlindedBeaconBlockBody{})
+)
+
+// bellatrixSignedBlindedBeaconBlock is a convenience wrapper around a Bellatrix beacon block
+// object. This wrapper allows us to conform to a common interface so that beacon
+// blocks for future forks can also be applied across prysm without issues.
+type bellatrixSignedBlindedBeaconBlock struct {
```

Comment:
All other files have `Bellatrix` as a suffix in object names. This file's types have `bellatrix` as a prefix. It might be good to settle for one option.